### ISSUE TITLE
Antialiasing is broken in song select

### ISSUE THREAD

**Issue by Lolihunter1337** [ppy/osu#9326]
**Describe the bug:** 
in 2020.616.0 Song select seems to appear normal. however in 2020.620.0 it seems to be broken?
**Screenshots or videos showing encountered issue:** 
![image](https://user-images.githubusercontent.com/17437302/85208293-5c347d00-b32f-11ea-9e8f-664c928c308a.png)

![image](https://user-images.githubusercontent.com/17437302/85208271-35764680-b32f-11ea-89e1-d2abb5bed054.png)

**osu!lazer version:** 
2020.620.0
**Logs:**
[network.log](https://github.com/ppy/osu/files/4808295/network.log)
[performance.log](https://github.com/ppy/osu/files/4808296/performance.log)
[runtime.log](https://github.com/ppy/osu/files/4808297/runtime.log)
[updater.log](https://github.com/ppy/osu/files/4808298/updater.log)
[database.log](https://github.com/ppy/osu/files/4808299/database.log)

**Comment by bdach** [comment]
Can't reproduce this on my Linux setup, for whatever that's worth.

**Comment by MiraiSubject** [comment]
I can reproduce on current head.
![image](https://user-images.githubusercontent.com/803255/85209212-5b531980-b336-11ea-8b88-ab0a041752ea.png)

With the latest framework version on old builds I get the same issue.
So I just ` dotnet restore`'d for all the older tags until I found a build that doesn't have this aliasing problem. 

I found that ` 2020.613.0` (with the accompanied framework version) is the last version that does not have jagged edges in that area.
